src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
   margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+
+  --nav-list-color: #00acdc;
+}
+
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+}
+
+.nav_list ul {
+  display: flex;
+
+  list-style: none;
+  list-style-type: none;
+}
+
+.nav_list li a {
+  display: block;
+  line-height: 60px;
+}
+
+.nav_list li:not(:first-child) {
+  padding-left: 20px;
+}
+
+.nav_list ul li:nth-child(4) a:hover {
```

Comment:
Using `nth-child(4)` for styling a specific list item can be fragile if the list structure changes. Consider using a class to target this element instead for more maintainable code.